### PR TITLE
Fix introduction of statements and expressions

### PR DESCRIPTION
**Pull request by Dedekind561**
## What does this change?

Module: JS1 
Week(s):1

#249 

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

## Description

Fix the introduction of statements and declarations in week 1 of JavaScript.

### CODE REVIEW

File: content/js1/blocks/repl/index.md [hunk 1]
```diff
@@ -40,11 +40,11 @@ We input JavaScript instructions that are then executed by NodeJS. The REPL repl
 Type each of the following expressions [into the REPL](#repl-1-1) one at a time and then press enter to check the result.
 
 ```bash
-10 + 32;
```
Comment (Dedekind561) [inline content/js1/blocks/repl/index.md:43]: These were technically expression statements before and not expressions 😁

File: content/js1/blocks/scripts/index.md [hunk 2]
```diff
@@ -14,25 +14,30 @@ But usually, our programs have many instructions, and we want to keep and re-run
...
 
 1. In your terminal, create a new file called `example.js`.
-2. Try writing a few expressions in the file.
+2. Try writing a few lines in the file.
 3. Get Node to run this file. (Don't use the REPL now - you should run a command to execute the whole file.)
 
 {{</note>}}
 
-Once the computer evaluates these expressions, the execution of the program is complete. But we’re left with a problem. With the REPL, the user inputs an expression, the computer evaluates it and then prints the result. But now the computer will go and execute each line sequentially until completion _without_ printing the values of each expression it evaluates.
+Once the computer executes these statements, the execution of the program is complete. But we’re left with a problem. With the REPL, the user inputs an expression statement or declaration. Then the computer reads and executes the line and then immediately prints feedback to the terminal.
```
Comment (SallyMcGrath) [inline content/js1/blocks/scripts/index.md:38]:
```suggestion
Once the computer executes these statements, the execution of the program is complete. But we’re left with a problem. With the REPL, the user inputs an expression statement or declaration; the computer reads and executes the line and _immediately_ prints feedback to the terminal. With a file, the computer will execute each line sequentially until completion _without_ printing the values of each expression it evaluates.
```

File: content/js1/blocks/scripts/index.md [hunk 3]
```diff
@@ -14,25 +14,30 @@ But usually, our programs have many instructions, and we want to keep and re-run
...
+
+But with a file, the computer will execute each line sequentially until completion _without_ printing the values of each expression it evaluates.
```
Comment (SallyMcGrath) [inline content/js1/blocks/scripts/index.md:40]:
```suggestion

```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
yes but this is a bit clearer

**Pull request merged**